eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -68,7 +68,7 @@ transaction_body =
   , ? 17 : coin                     ; total collateral
   , ? 18 : set<transaction_input>   ; reference inputs
```

Comment:
Can you add a comment on the fields that you adjusted which are `<set>`s, that they can't be empty. I am not sure if there is a way to specify it with cddl.

I suspect we might have problems with CDDL tests because of this  :disappointed:

Reply:
Added a `nonempty_set<a> = [+ a]` to `cddl-files/mock/extras.cddl` and a corresponding definition to `cddl-files/real/extras.cddl` as well. I also increased the test runs to `10` from `5`, hoping that at least some more base cases could be covered.